### PR TITLE
Styling improvements to Ask Form

### PR DESCRIPTION
**Pull request by isaacdurazo**
@iros, @kadamwhite what do you think about this solution?
ask-form

### CODE REVIEW

File: src/components/EmbeddedAskForm.scss [hunk 1]
```diff
@@ -1,46 +1,43 @@
...
   padding: 1em;
+}
 
-  &.open {
-    .form-toggle-button {
-      background: #d9d9d9; // grey
-    }
-    .embedded-form {
-      transition: max-height 1s;
+.form-container.open {
```
Comment (kadamwhite) [inline src/components/EmbeddedAskForm.scss:11]: I'm going to merge this, but I'd ask to try to use the nesting SASS syntax in the future -- it's generally in use throughout and the `&.class` syntax and nesting helps me clarify the hierarchy.

File: src/components/EmbeddedAskForm.scss [hunk 2]
```diff
@@ -103,3 +100,36 @@
...
+
+
+@media only screen and (min-width: 25em) {
+  #MultipleChoice--2 label { 
+    width: initial !important;
+  }
+}
+
+@media only screen and (min-width: 25em) {
+  .form-container header, #ask-form > div > div > div {
```
Comment (kadamwhite) [inline src/components/EmbeddedAskForm.scss:117]: Eek! XD hard to style...

**Review by kadamwhite — APPROVED** [review]

**Comment by kadamwhite** [comment]
@isaacdurazo I think I like this! I'll let @iros weigh in in case there's any future adjustments needed, but notwithstanding the comments above I'm going to merge.

File: src/components/EmbeddedAskForm.scss (hunk 2 above)
Comment (isaacdurazo) [reply src/components/EmbeddedAskForm.scss:117]: Yah man, I was gonna talk to you about this in the AM. I feel we can improve the markup a lot and make things easier in the CSS side.

**Pull request merged**